app/globals.css

```
@@ -281,4 +319,212 @@
+  }
+
+  .piece-tile {
+    position: absolute;
+    top: 0;
+    left: 0;
+    --c: var(--c-2);
+    --r: var(--r-2);
+    --cs: var(--cs-2);
+    --rs: var(--rs-2);
+    --ar: var(--ar-2);
+    width: calc(var(--cs) * var(--col) + (var(--cs) - 1) * var(--grid-gap));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)